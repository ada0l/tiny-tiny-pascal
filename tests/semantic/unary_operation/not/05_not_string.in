var a: string;
begin writeln(nota); end.