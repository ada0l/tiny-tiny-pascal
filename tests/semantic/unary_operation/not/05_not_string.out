(2, 15): Unary operator is not overloaded not string